stories/imdb/components/IMDBCards.js

```diff
@@ -1,21 +0,0 @@
-import _ from 'lodash/fp'
-import React from 'react'
-import Flex from '../../../src/layout/Flex'
-import { contexturify } from '../../../src/utils/hoc'
-
-let IMDBCards = contexturify(({ node }) => (
```

Comment:
The design goal here was to communicate that you could have alternate UIs besides ResultsTable (and can even reuse pieces like results pager). ResultsTable is very robust, but you aren't required to use it. Let's make sure this still has a home somewhere

Reply:
a new and improved home 🙂

![results2](https://user-images.githubusercontent.com/35466670/64305019-f58ad200-cf5b-11e9-90ef-5e6017b7aedb.gif)